devenv.nix

```diff
@@ -286,4 +287,55 @@ in {
   scripts.debug.exec = ''
     XDEBUG_SESSION=1 ${phpXdebug}/bin/php "$@"
   '';
+
+  scripts.importdb.exec = ''
```

Comment:
could you move the content to the let part of this file?

Reply:
👍 Sure! changed!